enterprise/frontend/src/embedding-sdk/components/private/SdkContentWrapper.tsx

```diff
@@ -18,8 +27,39 @@ export const SdkContentWrapper = styled.div<{ font: string; baseUrl?: string }>`
   ${({ theme }) => getRootStyle(theme)}
 
   ${({ baseUrl }) => defaultFontFiles({ baseUrl })}
+  ${({ fontFiles }) =>
+    fontFiles?.map(
+      file => css`
+        @font-face {
+          font-family: "Custom";
+          src: url(${encodeURI(file.src)}) format("${file.fontFormat}");
+          font-weight: ${file.fontWeight};
+          font-style: normal;
+          font-display: swap;
+        }
+      `,
+    )}
 
   svg {
     display: inline;
   }
 `;
+interface SdkContentWrapperProps {
+  font: string;
+  baseUrl?: string;
+}
+export function SdkContentWrapper({
+  font,
+  baseUrl,
+  ...divProps
+}: SdkContentWrapperProps & HTMLAttributes<HTMLDivElement>) {
+  const fontFiles = useSelector(getFontFiles);
```

Comment:
Since I need to access a selector I renamed the old component and create a new one with the same name.